Ports/RISC-V/RV32/GCC/os_cpu_a.S

```diff
@@ -262,9 +337,11 @@ Software_IRQHandler:
     addi   sp, sp, 4 * 32
 
 # Exception return will restore remaining context
+    
+    li t0, 0x80 #set MPIE = 1
```

Comment:
Put some more comment here like
setting MPIE = 1 will make sure when mret is called, interrupts are enabled